obspy/core/util/headers.py

```diff
@@ -0,0 +1,484 @@
+# -*- coding: utf-8 -*-
+"""
+Base constants for ObsPy.
+
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+:license:
+    GNU Lesser General Public License, Version 3
+    (https://www.gnu.org/copyleft/lesser.html)
+"""
+# NO IMPORTS FROM OBSPY OR FUTURE IN THIS FILE! (file gets used at
+# installation time)
+import sys
+
+
+# XXX TODO when bumping to numpy 1.9.0: replace bytes() in io.reftek with
+# np.tobytes()
+HARD_DEPENDENCIES = {
+    "future": ">=0.12.4",
+    "numpy": ">=1.6.1",
+    "scipy": ">=0.9.0",
+    "matplotlib": ">=1.1.0",
+    "lxml": "",
+    "setuptools": "",
+    "sqlalchemy": "",
+    "decorator": "",
+    "requests": ""
+    }
+OPTIONAL_DEPENDENCIES = {
+    'arclink': {
+        "cryptography": "",
+        "m2crypto": "",
+        "pycrypto": ""},
+    # for docs, also see conda environment file under
+    # misc/docs/py3-docs-master.yml
+    # other versions might work as well but these are currently used in our
+    # docs buildbot and therefore confirmed to work
+    'docs': {
+        "sphinx": "==1.2.2",
+        "sphinx-bootstrap-theme": "==0.4.8",
+        # for 'make coverage'
+        "coverage": "==4.0",
+        # for 'make citations'
+        "pybtex": "==0.19",
+        # for 'make docset'
+        "doc2dash": "==2.0.2",
+        # for tutorial
+        # https://github.com/obspy/wheelhouse/raw/master/
+        #      basemap-1.0.7-cp27-none-linux_x86_64.whl
+        # https://github.com/obspy/wheelhouse/raw/master/
+        #      mlpy-3.5.0-cp27-none-linux_x86_64.whl
+        "basemap": "",
+        "mlpy": "3.5.0",
+        # for doctests
+        "pyproj": ""},
+    'geodetics': {
+        "geographiclib": ""},
+    'imaging': {
+        "basemap": "",
+        "cartopy": ""},
+    'mseed': {
+        "jsonschema": ""},
+    'shapefile': {
+        "gdal": ""},
+    'tests': {
+        "flake8": ">=3.0",
+        "mock": "",  # Py2 only
+        "pep8-naming": "",
+        "pyflakes": "",
+        "pyimgur": "",
+        "pyproj": ""},
```

Comment:
Its only needed for the tests of a few specific modules. So maybe rather add it to those?